src/components/Transactions/Transactions.js

```diff
@@ -145,26 +168,14 @@ const Transactions = ({
       )}
     </WrapStyled>
   );
+};
 
 Transactions.propTypes = {
-  transactions: PropTypes.arrayOf(PropTypes.any),
-  error: PropTypes.arrayOf(PropTypes.any),
-  toggleTransactionsList: PropTypes.func.isRequired,
-  isTransactionsItemsLoading: PropTypes.bool,
-  isExpanded: PropTypes.bool,
-  t: PropTypes.func,
-  isShowMoreButtonHidden: PropTypes.bool,
-  isTransactionsSectionLoading: PropTypes.bool
+  t: PropTypes.func
 };
 
 Transactions.defaultProps = {
-  transactions: [],
-  error: [],
-  isTransactionsItemsLoading: false,
-  isExpanded: false,
-  t: k => k,
-  isShowMoreButtonHidden: false,
-  isTransactionsSectionLoading: false
+  t: k => k
 };
 
 export { Transactions as PureTransactions };
```

Comment:
`Transactions` component is also used as a standalone component. Please see the `package.js` file and see how Transactions are imported there (unfortunately its a duplication). Maybe we could get rid of this TransactionList.container and simply use this refactored component?